content/yaml/testing.md

```diff
@@ -50,5 +50,36 @@ scripts:
 
 ## Flutter drive test
 
-    flutter emulators --launch apple_ios_simulator                  # for android use: flutter emulators --launch emulator
-    flutter drive --target=test_driver/my_drive_target.dart
+#### On a mobile emulator
+
+        flutter emulators --launch apple_ios_simulator             # for android use: flutter emulators --launch emulator
+        flutter drive --target=test_driver/my_drive_target.dart
+
+#### Web on a browser
```

Comment:
I'd rephrase as: Running on a web browser
Also, should be ###

Reply:
Fixed, plus added that it's not actually run on browser, but browser driver